lib/src/isolate_channel.dart

```diff
@@ -93,7 +95,7 @@ class IsolateChannel<T> extends StreamChannelMixin<T> {
   factory IsolateChannel(ReceivePort receivePort, SendPort sendPort) {
```

Comment:
Can't type the argument as `ReceivePort<T>`?

Reply:
no, ReceivePort does not have a generic argument, every ReceivePort is a `Stream<dynamic>`.

Also, if we tried to get that changed in the SDK we could _maybe_ type this one (somewhat unsure of that) but we still wouldn't be able to force the other case to be `ReceivePort<T>` since the first thing that comes across it is a `SendPort` that we send.